lib/friendly_shipping/services/usps_ship/shipping_methods.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module FriendlyShipping
+  module Services
+    class USPSShip
+      CLASS_IDS = {
+        priority_mail_express: {
+          standard: '3',
+          hold_for_pickup: '2',
+          sunday_holiday_delivery: '23'
+        },
+        priority_mail_cubic: '999',
+        ground_advantage: '1058'
+      }.freeze
+
+      SHIPPING_METHODS = [
+        ['PARCEL_SELECT', 'Parcel Select'],
+        ['PARCEL_SELECT_LIGHTWEIGHT', 'Parcel Select Lightweight'],
+        ['PRIORITY_MAIL_EXPRESS', 'Priority Mail Express', CLASS_IDS[:priority_mail_express].values],
+        ['PRIORITY_MAIL', 'Priority Mail'],
+        ['FIRST-CLASS_PACKAGE_SERVICE', 'First-Class Package Service'],
+        ['LIBRARY_MAIL', 'Library Mail'],
+        ['MEDIA_MAIL', 'Media Mail'],
+        ['BOUND_PRINTED_MATTER', 'Bound Printed Matter'],
+        ['USPS_CONNECT_LOCAL', 'USPS Connect Local'],
+        ['USPS_CONNECT_MAIL', 'USPS Connect Mail'],
+        ['USPS_CONNECT_NEXT_DAY', 'USPS Connect Next Day'],
+        ['USPS_CONNECT_REGIONAL', 'USPS Connect Regional'],
+        ['USPS_CONNECT_SAME_DAY', 'USPS Connect Same Day'],
+        ['USPS_GROUND_ADVANTAGE', 'USPS Ground Advantage', CLASS_IDS[:ground_advantage]],
+        ['USPS_GROUND_ADVANTAGE_RETURN_SERVICE', 'USPS Ground Advantage Return Service'],
+        ['USPS_RETAIL_GROUND', 'USPS Retail Ground'],
+        ['GROUND_RETURN_SERVICE', 'Ground Return Service'],
+        ['FIRST-CLASS_PACKAGE_RETURN_SERVICE', 'First-Class Package Return Service'],
+        ['PRIORITY_MAIL_RETURN_SERVICE', 'Priority Mail Return Service'],
+        ['PRIORITY_MAIL_EXPRESS_RETURN_SERVICE', 'Priority Mail Express Return Service']
+        # TODO where is PRIORITY_MAIL_CUBIC?
+        # ['PRIORITY MAIL CUBIC', 'Priority Mail Cubic', CLASS_IDS[:priority_mail_cubic]],
```

Comment:
When does this need to be determined?

Reply:
Ah, good catch. I forgot about that. Let me do some digging...

Reply:
It appears the new API doesn't have a dedicated mail class for cubic. Instead, we have to pass `rateIndicator=CP` and `priceType=COMMERCIAL` to get the equivalent cubic pricing.

I'll work on updating the code accordingly and will re-request a review when done.

Reply:
We should be good to go now. Users will have to pass the correct rate indicator and price type in the options (along with the `PRIORITY_MAIL` mail type) to get cubic pricing.